Lib/fontTools/feaLib/builder.py

```diff
@@ -8,15 +8,19 @@
 import itertools
 
 
-def addOpenTypeFeatures(featurefile_path, font):
-    builder = Builder(featurefile_path, font)
+def addOpenTypeFeatures(font, featurefile_path=None, featurefile_data=None):
```

Comment:
wouldn't `feapath` and `feadata` be clear enough as keyword argument names here?
`featurefile_data` seems to me a bit too long/verbose...
/cc @adrientetar @brawer